desed: 1.2.1 -> 1.2.2

vinylen opened the pull request:
https://github.com/SoptikHa2/desed/releases/tag/v1.2.2

Bumped desed version from v1.2.1 to v1.2.2

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

liff committed:
ravedude: ensure avrdude is in the PATH

ravedude attempts to execute `avrdude` from the current `PATH` and
unless the user has separately installed `avrdude`, that will fail.

nagisa committed:
grafanaPlugins: modify the update script to work for all systems

This also now grabs checksums off the API which saves on downloads and
such.

DerDennisOP committed:
python312Packages.openai-triton: 2.1.0 -> 3.0.0, update links to repository

DerDennisOP committed:
python312Packages.triton*: repair package

SomeoneSerge committed:
python312Packages.triton: use more generic patch to unvendor ptxas/cuda

torgeirsh committed:
rdkafka: Add curl dependency for OAuth support.

jackwilsdon committed:
plexamp: add ozone platform hint for wayland

Stunkymonkey committed:
nixos/services.mysql: remove `with lib;`

r-vdp committed:
nixos/etc-overlay: avoid rebuilding the initrd every time the etc contents change

Before this change, the hash of the etc metadata image was included in
the mount unit that's responsible for mounting this metadata image in the
initrd.
And because this metadata image changes with every change to the etc
contents, the initrd would be rebuild every time as well.
This can lead to a lot of rebuilds (especially when revision info is
included in /etc/os-release) and all these initrd archives use up a lot of
space on the ESP.

With this change, we instead include a symlink to the metadata image in the
top-level directory, in the same way as we already do for things like init and
prepare-root, and we deduce the store path from the init= kernel parameter,
in the same way as we already do to find the path to init and prepare-root.

Doing so avoids rebuilding the initrd all the time.

chreekat committed:
nixos/github-runners: make enable functional

Fixes #305304

Aleksanaa committed:
gnome-graphs: 1.8.1 -> 1.8.2

SebTM committed:
proton-vpn-local-agent: init at 0-unstable-2024-10-10

Newly created library for the ProtonVPN application.

a-kenji committed:
cosmic-comp: 1.0.0-alpha.1 -> 1.0.0-alpha.2

a-kenji committed:
xdg-desktop-portal-cosmic: 1.0.0-alpha.1 -> 1.0.0-alpha.2

a-kenji committed:
xdg-desktop-portal-cosmic: copy icons, portal.conf to $out/share

a-kenji committed:
cosmic-settings-daemon: unstable-2023-12-29 -> 1.0.0-alpha.2

a-kenji committed:
cosmic-settings-daemon: inline pname repo

a-kenji committed:
cosmic-settings-daemon: install polkit rules

Commit:
nix-search-cli: 0-unstable-2023-09-12 -> 0.2-unstable-2024-09-24

nagisa committed:
grafanaPlugins.grafana-clickhouse-datasource: 3.3.0 -> 4.4.0

nagisa committed:
grafanaPlugins.grafana-clock-panel: 2.1.3 -> 2.1.8

nagisa committed:
grafanaPlugins.grafana-oncall-app: v1.8.5 -> 1.10.2

nagisa committed:
grafanaPlugins.grafana-polystat-panel: 2.1.4 -> 2.1.13

nagisa committed:
grafanaPlugins.bsull-console-datasource: init at 1.0.1

nagisa committed:
grafanaPlugins.fetzerch-sunandmoon-datasource: init at 0.3.3

nagisa committed:
grafanaPlugins.frser-sqlite-datasource: init at 3.5.0

nagisa committed:
grafanaPlugins.grafana-discourse-datasource: init at 2.0.2

nagisa committed:
grafanaPlugins.grafana-github-datasource: init at 1.9.0

nagisa committed:
grafanaPlugins.grafana-googlesheets-datasource: init at 1.2.14

nagisa committed:
grafanaPlugins.grafana-mqtt-datasource: init at 1.1.0-beta.2

nagisa committed:
grafanaPlugins.grafana-opensearch-datasource: init at 2.19.0

nagisa committed:
grafanaPlugins.marcusolsson-calendar-panel: init at 3.7.0

nagisa committed:
grafanaPlugins.marcusolsson-csv-datasource: init at 0.6.19

nagisa committed:
grafanaPlugins.marcusolsson-json-datasource: init at 1.3.17

nagisa committed:
grafanaPlugins.ventura-psychrometric-panel: init at 4.5.1

nagisa committed:
grafanaPlugins.volkovlabs-echarts-panel: init at 6.4.1

nagisa committed:
grafanaPlugins.volkovlabs-form-panel: init at 4.6.0

nagisa committed:
grafanaPlugins.volkovlabs-rss-datasource: init at 4.2.0

nagisa committed:
grafanaPlugins.volkovlabs-variable-panel: init at 3.5.0

nagisa committed:
grafanaPlugins.yesoreyeram-infinity-datasource: init at 2.11.0

r-ryantm committed:
museum: photos-v0.9.35 -> photos-v0.9.46

maxbrunet committed:
kubelogin: install shell completions

wentasah committed:
git-cliff: Add man page and shell completion

aanderse committed:
nixos/userborn: fix username typo

errnoh committed:
coroot: init at 1.5.8

vbgl committed:
ocamlPackages.merlin: 5.1-502 → 5.2.1-502

azuwis committed:
uxplay: correct nativeBuildInputs and buildInputs

azuwis committed:
uxplay: fix build on darwin

lukts30 committed:
write_stylus: rename to styluslabs-write-bin

r-vdp committed:
nixos/systemd-initrd: pull the logic to find the nixos closure into a separate service

vbgl committed:
ocamlPackages.pp: 1.2.0 → 2.0.0

FliegendeWurst committed:
ccid: 1.5.5 -> 1.6.1

markuskowa committed:
amd-libflame: 4.2 -> 5.0

markuskowa committed:
aocl-utils: 4.2 -> 5.0

markuskowa committed:
aocl-utils: apply nixfmt

markuskowa committed:
amd-libflame: apply nixfmt

r-ryantm committed:
jake: 10.8.7 -> 10.9.1

tobim committed:
google-cloud-cpp: enable darwin, fix cross

kirillrdy committed:
nixosTests.timescaledb: fix build, bump postgres version

tobim committed:
Revert "pkgsStatic.protobuf: fix installation of protoc"

This reverts commit 41963339b973da3e3782e227aee22e3537f21909.

paperdigits committed:
alglib: 4.02.0 -> 4.03.0

dotlambda committed:
python312Packages.mat2: disable failing test

r-ryantm committed:
nb: 7.12.1 -> 7.14.4

GaetanLepage committed:
python312Packages.imageio: 2.35.1 -> 2.36.0

Diff: https://github.com/imageio/imageio/compare/refs/tags/v2.35.1...v2.36.0

Changelog: https://github.com/imageio/imageio/blob/v2.36.0/CHANGELOG.md

Steinhagen committed:
lutris: fix libjansson and cursor issues

This fixes the following warning:
ntlm_auth: /home/reloner/.local/share/lutris/runtime/Ubuntu-18.04-x86_64/libjansson.so.4: no version information available (required by /nix/store/4gcggjvzrbywi92r66p6l15hckxng2q0-samba-4.20.1/lib/samba/libcommon-auth-private-samba.so)
The reason why this issue is happening is because the Lutris FHS that you are running doesn't install jansson as a dependency. By default the WoW entry tries to prioritize the System's libraries and if it doesn't find something, it will fallback to the Lutris runtime. Since the runtime version is really old, you'll see that error.

This commit also fixes cursor issues caused by xorg in WoW.

r-ryantm committed:
eask-cli: 0.10.0 -> 0.10.1

fabaff committed:
kics: 2.1.2 -> 2.1.3

Changelog: https://github.com/Checkmarx/kics/releases/tag/v2.1.3

fabaff committed:
gitxray: init at -1.0.15-unstable-2024-09-20

Tool which leverages Public GitHub REST APIs for various tasks

https://github.com/kulkansecurity/gitxray

GaetanLepage committed:
snakemake: move to by-name

GaetanLepage committed:
snakemake: format

GaetanLepage committed:
snakemake: 8.20.1 -> 8.23.0

Diff:
https://github.com/snakemake/snakemake/compare/refs/tags/v8.20.1...v8.23.0

Changelog:
https://github.com/snakemake/snakemake/blob/refs/tags/v8.23.0/CHANGELOG.md

GaetanLepage committed:
python312Packages.snakemake-interface-executor-plugins: 9.2.0 -> 9.3.2

Diff:
https://github.com/snakemake/snakemake-interface-executor-plugins/compare/refs/tags/v9.2.0...v9.3.2

Changelog:
https://github.com/snakemake/snakemake-interface-executor-plugins/blob/v9.3.2/CHANGELOG.md

GaetanLepage committed:
python312Packages.conda-inject: init at 1.3.2

Commit:
python3Packages.triton: fix cuda (ptxas, cudart paths)

Commit:
python3Packages.triton.tests.axpy-cuda: init

Pandapip1 committed:
localsend: remove absolute path in desktop entry

Addresses https://github.com/NixOS/nixpkgs/issues/308324

r-ryantm committed:
homebank: 5.8.3 -> 5.8.5

viraptor committed:
clfft: fix darwin build

Default to gcc on Darwin. There were no commits in a long time so no patches
are expected for the new Clang compatibility issue.

r-ryantm committed:
cimg: 3.4.2 -> 3.4.3

numinit committed:
mattermost: 9.5.10 -> 9.5.11

Fixes MMSA-2024-00370; see:
https://mattermost.com/security-updates/

numinit committed:
{mattermost,matterircd}: relocate to by-name

viraptor committed:
copycat: fix darwin build

Add the required framework.

r-ryantm committed:
nanopb: 0.4.8 -> 0.4.9

r-ryantm committed:
vpl-gpu-rt: 24.2.5 -> 24.3.3

u2d3rsc0re committed:
tlp: 1.6.1 -> 1.7.0

hellodword committed:
sing-box: 1.9.7 -> 1.10.0

r-ryantm committed:
marwaita-icons: 5.0 -> 5.1

paperdigits committed:
hdrmerge: unstable-2023-01-04 -> 0.5.0-unstable-2024-08-02

NickCao committed:
stratisd: 3.7.1 -> 3.7.2

Diff: https://github.com/stratis-storage/stratisd/compare/refs/tags/stratisd-v3.7.1...stratisd-v3.7.2

aqrln committed:
prisma-engines: 5.18.0 -> 5.21.0

bjornfor committed:
maven: 3.9.8 -> 3.9.9

NickCao commented on pkgs/by-name/de/desed/package.nix:
```diff
@@ -3,22 +3,21 @@
   fetchFromGitHub,
 }:
-
 rustPlatform.buildRustPackage {
```
```suggestion
rustPlatform.buildRustPackage rec {
```

NickCao commented on pkgs/by-name/de/desed/package.nix:
```diff
@@ -3,22 +3,21 @@
     repo = "desed";
-    rev = "master";
-    hash = "sha256-iCpEfefXXognk4YI1LLb3mwgaqMw4m3haq/gdS1JbQU=";
+    rev = "9dc58f95f49be8551001797f8d6065982f977fd7";
```
```suggestion
    rev = "refs/tags/v${version}";
```

NickCao commented on pkgs/by-name/de/desed/package.nix:
```diff
@@ -3,22 +3,21 @@
 
   meta = {
-    changelog = "https://github.com/SoptikHa2/desed/releases/tag/v1.2.1";
+    changelog = "https://github.com/SoptikHa2/desed/releases/tag/v1.2.2";
```
```suggestion
    changelog = "https://github.com/SoptikHa2/desed/releases/tag/v${version}";
```

aqrln committed:
prisma: 5.18.0 -> 5.21.0

liff committed:
ravedude: add liff as a maintainer

Pandapip1 committed:
localsend: Use copyDesktopItems

HeitorAugustoLN committed:
audacity: move to by-name

HeitorAugustoLN committed:
audacity: format with nixfmt-rfc-style

HeitorAugustoLN committed:
audacity: remove `with lib` from meta and `rec`

r-ryantm committed:
pinentry-rofi: 2.2.0 -> 3.0.0

vinylen commented:
Hey @NickCao thanks for your feedback! I've updated the PR include all of your suggested changes.

fabaff committed:
python312Packages.findimports: 2.5.0 -> 2.5.1

Diff: https://github.com/mgedmin/findimports/compare/refs/tags/2.5.0...2.5.1

Changelog: https://github.com/mgedmin/findimports/blob/2.5.1/CHANGES.rst

fabaff committed:
python312Packages.findimports: refactor

lukts30 committed:
styluslabs-write: init at 2024-10-12

r-ryantm committed:
python312Packages.cvelib: 1.5.0 -> 1.6.0

khaneliman committed:
cliphist: 0.5.0 -> 0.6.0

khaneliman committed:
cliphist: move to by-name

khaneliman committed:
cliphist: add update script

dotlambda committed:
immich: 1.117.0 -> 1.118.1

https://github.com/immich-app/immich/releases/tag/v1.118.0
https://github.com/immich-app/immich/releases/tag/v1.118.1

dotlambda committed:
nixos/immich: change default port to 2283

This was always upstream's default but they also change the internal
port, i.e. behind the reverse proxy, to 2283 in
https://github.com/immich-app/immich/pull/13185.

skeuchel committed:
alsa-firmware: fix riscv64-linux build

fabaff committed:
grimoire: init at 0.1.0

Tool to generate datasets of cloud audit logs for common attacks

https://github.com/DataDog/grimoire

dotlambda committed:
python312Packages.sense-energy: 0.12.4 -> 0.13.2

Diff: https://github.com/scottbonline/sense/compare/refs/tags/0.12.4...0.13.2

Changelog: https://github.com/scottbonline/sense/releases/tag/0.13.0
           https://github.com/scottbonline/sense/releases/tag/0.13.1
           https://github.com/scottbonline/sense/releases/tag/0.13.2

r-ryantm committed:
pv: 1.8.12 -> 1.8.14

GaetanLepage committed:
python312Packages.open-clip-torch: 2.26.1 -> 2.27.0

Diff: https://github.com/mlfoundations/open_clip/compare/refs/tags/v2.26.1...v2.27.0

Changelog: https://github.com/mlfoundations/open_clip/releases/tag/v2.27.0

GaetanLepage committed:
python312Packages.lightning-utilities: 0.11.7 -> 0.11.8

Diff: https://github.com/Lightning-AI/utilities/compare/refs/tags/v0.11.7...v0.11.8

Changelog: https://github.com/Lightning-AI/utilities/releases/tag/v0.11.8

dotlambda committed:
python312Packages.onnx: clean up dependencies

fabaff committed:
cilium-cli: 0.16.15 -> 0.16.19

Diff: https://github.com/cilium/cilium-cli/compare/refs/tags/v0.16.15...v0.16.19

Changelog: https://github.com/cilium/cilium-cli/releases/tag/v0.16.19

dotlambda committed:
python312Packages.onnx: 1.16.2 -> 1.17.0

Diff: https://github.com/onnx/onnx/compare/refs/tags/v1.16.2...v1.17.0

Changelog: https://github.com/onnx/onnx/releases/tag/v1.17.0

NickCao committed:
telegram-desktop: 5.6.2 -> 5.6.3

Diff: https://github.com/telegramdesktop/tdesktop/compare/f2e0e481de5d379b9542f6ff9021fab5d409c1d4...v5.6.3

Changelog: https://github.com/telegramdesktop/tdesktop/releases/tag/v5.6.3

fabaff committed:
python312Packages.tencentcloud-sdk-python: 3.0.1249 -> 3.0.1250

Diff: https://github.com/TencentCloud/tencentcloud-sdk-python/compare/refs/tags/3.0.1249...3.0.1250

Changelog: https://github.com/TencentCloud/tencentcloud-sdk-python/blob/3.0.1250/CHANGELOG.md

NickCao committed:
v2ray-domain-list-community: 20240920063125 -> 20241013063848

Diff: https://github.com/v2fly/domain-list-community/compare/20240920063125...20241013063848

fabaff committed:
python312Packages.opower: 0.8.3 -> 0.8.4

Diff: https://github.com/tronikos/opower/compare/refs/tags/v0.8.3...v0.8.4

Changelog: https://github.com/tronikos/opower/releases/tag/v0.8.4

fabaff committed:
python312Packages.pydeconz: 117 -> 118

Diff: https://github.com/Kane610/deconz/compare/refs/tags/v117...v118

Changelog: https://github.com/Kane610/deconz/releases/tag/v118

fabaff committed:
python312Packages.aioftp: 0.22.3 -> 0.23.1

Steinhagen committed:
heroic: fix icon sizes

By default, the 128x128 icon was installed as a 512x512.
The modification uses the svg icon present in the deployment to add the scalable one.

This should fix: https://github.com/NixOS/nixpkgs/issues/348764

r-ryantm committed:
kustomize: 5.4.3 -> 5.5.0

pyrox0 committed:
python312Packages.androguard: Remove unused dependencies

pyrox0 committed:
python312Packages.androguard: Remove mock dependency

pyrox0 committed:
python312Packages.androguard: Remove `with lib;`

khaneliman committed:
cliphist: 0.6.0 -> 0.6.1

donovanglover committed:
bluetui: init at 0.5.1

r-ryantm committed:
python312Packages.gcovr: 7.2 -> 8.2

jchv committed:
linuxPackages.lttng-modules: 2.13.13 -> 2.13.15

Fixes the build for linuxPackages_latest.lttng-modules.

jchv committed:
linuxPackages.lttng-modules: reformat with RFC style

r-ryantm committed:
python312Packages.stdlib-list: 0.10.0 -> 0.11.0

github-actions[bot] committed:
terraform-providers.helm: 2.15.0 -> 2.16.1

github-actions[bot] committed:
terraform-providers.huaweicloud: 1.68.1 -> 1.69.0

github-actions[bot] committed:
terraform-providers.libvirt: 0.7.6 -> 0.8.0

github-actions[bot] committed:
terraform-providers.kubernetes: 2.32.0 -> 2.33.0

github-actions[bot] committed:
terraform-providers.ibm: 1.69.0 -> 1.70.0

github-actions[bot] committed:
terraform-providers.linode: 2.27.0 -> 2.29.1

github-actions[bot] committed:
terraform-providers.local: 2.5.1 -> 2.5.2

github-actions[bot] committed:
terraform-providers.minio: 2.5.0 -> 2.5.1

github-actions[bot] committed:
terraform-providers.lxd: 2.3.0 -> 2.4.0

github-actions[bot] committed:
terraform-providers.migadu: 2024.9.5 -> 2024.10.10

github-actions[bot] committed:
terraform-providers.mongodbatlas: 1.18.1 -> 1.21.1

github-actions[bot] committed:
terraform-providers.newrelic: 3.45.0 -> 3.50.0

github-actions[bot] committed:
terraform-providers.nomad: 2.3.1 -> 2.4.0

github-actions[bot] committed:
terraform-providers.ns1: 2.4.1 -> 2.4.4

github-actions[bot] committed:
terraform-providers.null: 3.2.2 -> 3.2.3

github-actions[bot] committed:
terraform-providers.openstack: 2.1.0 -> 3.0.0

github-actions[bot] committed:
terraform-providers.okta: 4.10.0 -> 4.11.0

github-actions[bot] committed:
terraform-providers.opentelekomcloud: 1.36.18 -> 1.36.20

github-actions[bot] committed:
terraform-providers.oci: 6.9.0 -> 6.13.0

github-actions[bot] committed:
terraform-providers.ovh: 0.48.0 -> 0.51.0

github-actions[bot] committed:
terraform-providers.pagerduty: 3.15.6 -> 3.16.0

r-ryantm committed:
snips-sh: 0.4.0 -> 0.4.1

github-actions[bot] committed:
terraform-providers.project: 1.7.2 -> 1.8.0

github-actions[bot] committed:
terraform-providers.random: 3.6.2 -> 3.6.3

github-actions[bot] committed:
terraform-providers.rundeck: 0.4.7 -> 0.4.9

github-actions[bot] committed:
terraform-providers.rancher2: 5.0.0 -> 5.1.0

github-actions[bot] committed:
terraform-providers.scaleway: 2.44.0 -> 2.46.0

github-actions[bot] committed:
terraform-providers.selectel: 5.3.0 -> 5.4.0

github-actions[bot] committed:
terraform-providers.spacelift: 1.15.0 -> 1.16.1

github-actions[bot] committed:
terraform-providers.spotinst: 1.190.0 -> 1.195.0

github-actions[bot] committed:
terraform-providers.snowflake: 0.95.0 -> 0.97.0

github-actions[bot] committed:
terraform-providers.tailscale: 0.16.2 -> 0.17.2

github-actions[bot] committed:
terraform-providers.sumologic: 2.31.3 -> 2.31.5

github-actions[bot] committed:
terraform-providers.temporalcloud: 0.0.11 -> 0.0.13

github-actions[bot] committed:
terraform-providers.talos: 0.5.0 -> 0.6.0

github-actions[bot] committed:
terraform-providers.tfe: 0.58.1 -> 0.59.0

github-actions[bot] committed:
terraform-providers.time: 0.12.0 -> 0.12.1

github-actions[bot] committed:
terraform-providers.tencentcloud: 1.81.120 -> 1.81.133

github-actions[bot] committed:
terraform-providers.tls: 4.0.5 -> 4.0.6

github-actions[bot] committed:
terraform-providers.turbot: 1.11.1 -> 1.11.2

github-actions[bot] committed:
terraform-providers.vcd: 3.13.0 -> 3.14.0

github-actions[bot] committed:
terraform-providers.venafi: 0.21.0 -> 0.21.1

github-actions[bot] committed:
terraform-providers.vsphere: 2.9.1 -> 2.9.3

github-actions[bot] committed:
terraform-providers.yandex: 0.128.0 -> 0.130.0

r-ryantm committed:
vrc-get: 1.8.1 -> 1.8.2

zowoq committed:
Revert "terraform-providers: bump go to 1.23"

1.23 is now the default

This reverts commit d67593d858e1f759246089cde74592773974578e.

zowoq committed:
.github/workflows: remove update-terraform-providers

semi-broken, will try using r-ryantm bot for updates instead

zowoq committed:
terraform-providers: enable update bot

r-ryantm committed:
tdnf: 3.5.8 -> 3.5.9

r-ryantm committed:
parallel: 20240722 -> 20240922

tomodachi94 committed:
{nodePackages,vimPlugins}.coc-python: drop

See notice in the README:
https://github.com/neoclide/coc-python

> WARNING: it's recommended to use coc-pyright if
> you're using python3 or use coc-jedi if you're using jedi,
> the code of coc-python is too hard to maintain!

If that isn't convincing, the repo was archived on 2020-12-24.

Part of #229475

iynaix committed:
wallust: move to by-name

iynaix committed:
wallust: 3.0.0 -> 3.1.0

Changelog: https://codeberg.org/explosion-mental/wallust/releases/tag/3.1.0
Diff: https://codeberg.org/explosion-mental/wallust/compare/3.0.0...3.1.0

K900 committed:
buildFHSEnv: don't include /var

This mostly prevents random junk from leaking into fhsenvs,
but also fixes Steam not having a /var anymore.

fabaff committed:
python312Packages.gcovr: refactor

GaetanLepage committed:
python312Packages.onnx: 1.16.2 -> 1.17.0

Diff: https://github.com/onnx/onnx/compare/refs/tags/v1.16.2...v1.17.0

Changelog: https://github.com/onnx/onnx/releases/tag/v1.17.0

msanft committed:
erofs-utils: 1.8.1 -> 1.8.2

r-ryantm committed:
tiny-dfr: 0.3.1 -> 0.3.2

r-ryantm committed:
soju: 0.8.1 -> 0.8.2

VergeDX committed:
qv2ray: fix build

GaetanLepage committed:
python312Packages.nvidia-ml-py: fix and improve test

Aleksanaa committed:
gitmux: fix build

kachick committed:
sequoia-chameleon-gnupg: unstable-2023-11-22 -> 0.11.2

r-ryantm committed:
python312Packages.pyoverkiz: 1.14.1 -> 1.14.2

r-ryantm committed:
python312Packages.libknot: 3.4.0 -> 3.4.1

r-ryantm committed:
seafile-shared: 9.0.7 -> 9.0.8

fabaff committed:
python312Packages.tencentcloud-sdk-python: 3.0.1250 -> 3.0.1251

Diff: https://github.com/TencentCloud/tencentcloud-sdk-python/compare/refs/tags/3.0.1250...3.0.1251

Changelog: https://github.com/TencentCloud/tencentcloud-sdk-python/blob/3.0.1251/CHANGELOG.md

GaetanLepage committed:
python312Packages.kserve: 0.13.1 -> 0.14.0

Diff: https://github.com/kserve/kserve/compare/refs/tags/v0.13.1...v0.14.0

Changelog: https://github.com/kserve/kserve/releases/tag/v0.14.0

r-ryantm committed:
tbls: 1.77.0 -> 1.78.0

r-ryantm committed:
cargo-deb: 2.5.1 -> 2.7.0

r-ryantm committed:
libcdada: 0.6.0 -> 0.6.1

fabaff committed:
cnspec: 11.25.0 -> 11.26.0

Diff: https://github.com/mondoohq/cnspec/compare/refs/tags/v11.25.0...v11.26.0

Changelog: https://github.com/mondoohq/cnspec/releases/tag/v11.26.0

jwillikers committed:
nushell: 0.98.0 -> 0.99.0

changelog: https://www.nushell.sh/blog/2024-10-15-nushell_0_99_0.html

mweinelt committed:
prometheus-knot-exporter: 3.4.0 -> 3.4.1

hellodword committed:
nixos/sing-box: migration of deprecated fields

4evy committed:
google-chrome: fix URL encoding for version filter

4evy committed:
google-chrome: add `shellcheck`

4evy committed:
google-chrome: 129.0.6668.100 -> 130.0.6723.58

Changelog: https://chromereleases.googleblog.com/2024/10/stable-channel-update-for-desktop_15.html

4evy committed:
google-chrome: 129.0.6668.101 -> 130.0.6723.59

Changelog: https://chromereleases.googleblog.com/2024/10/stable-channel-update-for-desktop_15.html

mweinelt committed:
epshome: 2024.9.2 -> 2024.10.0

https://github.com/esphome/esphome/releases/tag/2024.10.0

r-ryantm committed:
vscode-extensions.pylyzer.pylyzer: 0.1.8 -> 0.1.10

fabaff committed:
python312Packages.pyoverkiz: update disabled

corngood committed:
jam: fix OS determination on linux

corngood committed:
jam: add test for OS=UNKNOWN

hellodword committed:
nixos/sing-box: test iproute2_table_index and iproute2_rule_index

Commit:
triton-llvm: patch for glibc 2.40 support

ryan4yin committed:
tiled: add qtwayland to buildInputs

fabaff committed:
python312Packages.python-toolbox: 1.0.10 -> 1.0.11

Diff: https://github.com/cool-RR/python_toolbox/compare/refs/tags/1.0.10...1.0.11

Changelog: https://github.com/cool-RR/python_toolbox/releases/tag/1.0.11

r-ryantm committed:
mlx42: 2.3.4 -> 2.4.0

khaneliman committed:
gh: format

khaneliman committed:
gh: 2.58.0 -> 2.59.0

Artturin committed:
vintagestory: Fix cursor on wayland

Capturing and Texture.

Doesn't need to use `LD_PRELOAD`, `LD_LIBRARY_PATH` works.

https://www.github.com/NixOS/nixpkgs/issues/265817#issuecomment-2345008152

fabaff committed:
python312Packages.python-ipware: 2.0.0 -> 3.0.0

Diff: https://github.com/un33k/python-ipware/compare/refs/tags/v2.0.0...v3.0.0

Changelog: https://github.com/un33k/python-ipware/blob/v3.0.0/CHANGELOG.md

r-vdp committed:
nixos/etc-overlay: fix VM test for immutable overlay

techknowlogick committed:
terraform: 1.9.7 -> 1.9.8

Diff: https://github.com/hashicorp/terraform/compare/v1.9.7...v1.9.8

Changelog: https://github.com/hashicorp/terraform/blob/v1.9.8/CHANGELOG.md

K900 committed:
obs-vkcapture: don't depend on ECM

Fixes #349078.

r-ryantm committed:
python312Packages.latexrestricted: 0.4.0 -> 0.5.0

adamcstephens committed:
nixos/atticd: wants network-online.target

fixes:

trace: evaluation warning: atticd.service is ordered after 'network-online.target' but doesn't depend on it

Mic92 committed:
nixos/test-driver: fix resource cleanup of vlan/qmp objects

Using __del__ is somewhat unsound resource cleanup in our clase the
logger already closed its logfile and therefor fails with exception
before the rest of the resources can be cleaned up.

r-ryantm committed:
python312Packages.shiv: 1.0.6 -> 1.0.7

NickCao commented:
Oh but we are getting some hash mismatch:
```
error: hash mismatch in fixed-output derivation '/nix/store/k0k7fz3wzc0hhrsvxw754g9ngd1nr0zg-source.drv':
         specified: sha256-k844tZgZmT1Sx2GNf8nb8HBnMzS59GS+MgU37KoKQaU=
            got:    sha256-aKkOs8IhnHjoJkXq9ryGn9fN0AmZyVTHbD/Vano+Erw=
```

alyssais committed:
cosmic-comp: install default config files

There's a Makefile that installs the Rust binary and some default
config files.

r-ryantm committed:
sqlitebrowser: 3.13.0 -> 3.13.1

dotlambda committed:
immich: add maintainers

zakame committed:
perlPackages.Appcpm: 0.997014 -> 0.997018

kachick committed:
cargo-make: fix shell completion

bbigras committed:
zed-editor: 0.156.2 -> 0.157.5

zakame committed:
perlPackages.XSParseSublike: 0.20 -> 0.29

ryand56 committed:
cinny-unwrapped, cinny-desktop: 4.2.1 -> 4.2.2

K900 committed:
steam (and friends): migrate to by-name, small cleanups all over

- rename "steam-original" or "steam" to "steam-unwrapped", as that's what it is
- rename "steam-fhsenv" to "steam", as that's what you actually want
- remove some no-longer-relevant hacks

a-kenji commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 348746`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>desed</li>
  </ul>
</details>

a-kenji commented on pkgs/by-name/de/desed/package.nix:
```diff
@@ -3,22 +3,21 @@
-    rev = "master";
-    hash = "sha256-iCpEfefXXognk4YI1LLb3mwgaqMw4m3haq/gdS1JbQU=";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-k844tZgZmT1Sx2GNf8nb8HBnMzS59GS+MgU37KoKQaU=";
```
```suggestion
    hash = "sha256-aKkOs8IhnHjoJkXq9ryGn9fN0AmZyVTHbD/Vano+Erw=";
```

a-kenji commented on pkgs/by-name/de/desed/package.nix:
```diff
@@ -3,22 +3,21 @@
   };
 
-  cargoHash = "sha256-z2qv394C0GhQC21HuLyvlNjrM65KFEZh1XLj+Y/B9ZM=";
+  cargoHash = "sha256-g9vc7FBiFRd9tS72YdL6IQ3dTYpyXnaND7MchO/Gfgg=";
```
```suggestion
  cargoHash = "sha256-3bkNirrvBOqIOhxD5xf3XsdaUMWFVT/lnN3gmZQIB9I=";
```

svrana committed:
juicefs: fix build

Resolves #349073

K900 committed:
kdePackages: Plasma 6.2.1 -> 6.2.1.1

Hot fixes, come get your piping hot fixes

r-ryantm committed:
abcmidi: 2024.08.13 -> 2024.10.10

r-ryantm committed:
ayatana-ido: 0.10.3 -> 0.10.4 (#349133)

dotlambda committed:
home-assistant-custom-components.homematicip_local: 1.65.0 -> 1.67.0

Diff: https://github.com/danielperna84/custom_homematic/compare/refs/tags/1.65.0...1.67.0

Changelog: https://github.com/danielperna84/custom_homematic/blob/1.67.0/changelog.md

dotlambda committed:
python312Packages.hahomematic: 2024.8.13 -> 2024.10.8

Diff: https://github.com/danielperna84/hahomematic/compare/refs/tags/2024.8.13...2024.10.8

Changelog: https://github.com/danielperna84/hahomematic/blob/refs/tags/2024.10.8/changelog.md

zowoq committed:
terraform-providers.cloudflare: 4.41.0 -> 4.44.0

techknowlogick committed:
terraform-providers.gitea: init at 0.5.1

vinylen commented:
This PR has been completely broken by me, i apologize in advance to anyone that has gotten a notification from this.
